test/unittest/unittest_nntrainer_modelfile.cpp

```diff
@@ -592,40 +579,37 @@ TEST(nntrainerIniTest, backbone_07_p) {
 
 /**
  * @brief Ini file unittest with backbone with normal backbone
- * @todo Fail at NN.loadFromConfig(s.getIniName())
  */
-// TEST(nntrainerIniTest, backbone_08_n) {
-//   ScopedIni s("backbone_08_n", {nw_base_mse, adam,
-//   backbone_random_external});
+TEST(nntrainerIniTest, backbone_08_n) {
+  ScopedIni s("backbone_08_n", {nw_base_mse, adam, backbone_random_external});
 
-//   nntrainer::NeuralNetwork NN;
+  nntrainer::NeuralNetwork NN;
 
-// #if defined(ENABLE_NNSTREAMER_BACKBONE) || defined(ENABLE_TFLITE_BACKBONE)
-//   EXPECT_EQ(NN.loadFromConfig(s.getIniName()), ML_ERROR_NONE);
-//   EXPECT_EQ(NN.compile(), ML_ERROR_INVALID_PARAMETER);
-//   EXPECT_EQ(NN.initialize(), ML_ERROR_NOT_SUPPORTED);
-// #else
-//   EXPECT_EQ(NN.loadFromConfig(s.getIniName()), ML_ERROR_NOT_SUPPORTED);
-// #endif
-// }
+#if defined(ENABLE_NNSTREAMER_BACKBONE) || defined(ENABLE_TFLITE_BACKBONE)
+  EXPECT_EQ(NN.loadFromConfig(s.getIniName()), ML_ERROR_INVALID_PARAMETER);
```

Comment:
It seems backbone_random_external from ScopdeIni is simply random.tflite; if ENABLE_TFLITE_BACKBONE is defined, shouldn't it expect ML_ERROR_NONE?

Reply:
You are correct, but we are already checking such case with `backbone_valid_external` from another TC, and I could not find any file for formulating `random.tflite` currently. I thought it is just a negative TC to emit error like this.